markdown/bugs/index.md

```diff
@@ -10,36 +10,21 @@ For troubleshooting the Iris installation, see the [Troubleshooting page][downlo
-
-  * Clicking "Stop" or "Dismiss" while calculating the fit statistics in either the "Fit" or "Confidence" tab will disconnect Sherpa from the SAMP hub. This prevents any more fitting routines within Iris. You can reconnect Sherpa to Iris from the command line:
-(in a new Terminal window or tab)
-
-		% /bin/bash       # if in a C-shell
-		$ source activate iris
-		$ sherpa-samp
-  
-    Wait 10 seconds for Sherpa to reconnect, then you may continue you fitting session. 
-
-  * When a SED is edited (e.g. by adding or removing segments) during a fitting session, the fitting window needs to be closed and the fitting session restarted from scratch. This is due to a bug in the fitting component. A dialog box will warn the user that if they change an SED during a fitting session, they should click OK when asked to close the fitting window itself. Clicking "Cancel" would leave the fitting manager in an undesired state, and out of sync with the rest of the Iris components.
-
-    **WORKAROUND:** If this happens, users can save the fitting session, close the fitting window, and then load the saved file in a new session. Due to the nature of the bug, no warnings can be displayed when the user starts the fitting tool but edits the SED before performing any actual fit.
-
-  * The Fitting Tool help bar is disabled. This is because the fitting tool help docs built in the Fitting Tool are for Specview; there is no information on fitting specifically in Iris. So to not confuse the readers with Specview-only operations, the help tool is left disabled.
+  * If you have model components with the same component IDs as those in a JSON file of a previously saved fit, loading that JSON file will override the current model components. If you load the JSON file first, then add new model components, the new model components will override the JSON model components if the model components share the same IDs.
 
 #### Fitting Data with Custom Models
 
   * When a spectral model component is deleted from the Custom Model Manager, the Sherpa model expression being used for fitting, shown in the Fitting Tool window, is not automatically updated; it needs to be edited manually by the user. 
 
-  * When fitting with a template model, the search method is automatically switched to a grid search over the templates included in the model. The template parameters are interpolated using the default K-nearest neighbor (KNN) algorithm in Sherpa 4.7, Iris v2.1's fitting engine. 
+  * When fitting with a template model, the search method is automatically switched to a grid search over the templates included in the model. The template parameters are interpolated using the default K-nearest neighbor (KNN) algorithm in Sherpa, Iris' fitting engine. 
```

Comment:
Hmm... was this included in 2.1 as well? The statement is contradicting itself, because no interpolation (of parameters) happens when the search method is grid search. Also, users need to provide the grid of values in grid search, and in any case grid search is not offered to users as an option in Iris, only the regular optimizers are. Rather, what happens in Iris is that templates are always interpolated. In other terms, the second sentence is correct, the first one is not.